### PR TITLE
Send message for members

### PR DESCRIPTION
**Pull request by diogomdp**
Members can also send a simple message
After clicking Run, you will see the name of the channel that the message was sent and the name of the server

### CODE REVIEW

**Review by almeidx — CHANGES_REQUESTED** [review]
The file has no extension.

**Comment by diogomdp** [comment]
placed extension : )

File: extensions/dbm_dashboard/dbm_dashboard_EXT/actions/actions/send_msg_members.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+module.exports = {
+    //----------------------------------------------------------------------------------
+    // Used to set the name of the mod / extension. 
+    // Note if this is an extension it cant have a space or it will not work.
+    name: "Send Menssage for channel",
```
Comment (almeidx) [inline extensions/dbm_dashboard/dbm_dashboard_EXT/actions/actions/send_msg_members.js]:
```suggestion
    name: "Send Message for channel",
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: extensions/dbm_dashboard/dbm_dashboard_EXT/actions/actions/send_msg_members.js (hunk 1 above)
Comment (diogomdp) [reply extensions/dbm_dashboard/dbm_dashboard_EXT/actions/actions/send_msg_members.js]: sorry 😕

**Review by almeidx — APPROVED** [review]

**Pull request merged**